lib/server.js

```diff
@@ -164,6 +184,59 @@ module.exports = function createServer(config, log) {
   )
 
   api.post(
+    '/checkAuthenticated',
+    function (req, res, next) {
+      var ip = req.body.ip
+      var token = req.body.token
```

Comment:
If you don't need the token for anything, don't pass it to customs-server here - the less often we send tokens over the wire, the less likely we are to accidentally leak them somewhere they shouldn't go.